src/hdusd/bl_nodes/nodes/texture.py

```diff
@@ -12,12 +12,42 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #********************************************************************
+import os
+
 from ..node_parser import NodeParser
+from . import log
 
 
 class ShaderNodeTexImage(NodeParser):
     def export(self):
-        return None
+        image_error_result = self.node_item((1.0, 0.0, 1.0, 1.0))
+        image = self.node.image
+        if not image:
+            log.warn(f"No image provided")
+            return image_error_result
+        # TODO support SEQUENCE image type
+        if image.source != 'FILE':
```

Comment:
pixels can be exported to temporary image file like .exr. Look as example into export.material.sync_mx()